Services/DataBaseService/Repositories/UserCredentialRepository.cs

```diff
@@ -0,0 +1,35 @@
+﻿using DataBaseService.DbModels;
+using DataBaseService.Interfaces;
+using DTO;
+using Microsoft.Data.SqlClient;
+using System;
+
+namespace DataBaseService.Repositories
+{
+    public class UserCredentialRepository : IRepository<UserEmailPassword>
+    {
+        private readonly IMapper<UserEmailPassword, DbUserCredential> mapper;
+        private readonly DataBaseContext dbContext;        
+        
+        public UserCredentialRepository(IMapper<UserEmailPassword, DbUserCredential> mapper, DataBaseContext dbContext)
+        {
+            this.mapper = mapper;
+            this.dbContext = dbContext;
+        }
+        public void Create(UserEmailPassword data)
```

Comment:
No empty line between constructor and method

Reply:
add line